### DISCUSSION TITLE
Anyone working on Laravel 9 compatibility?

### DISCUSSION THREAD

**Discussion by stefanzweifel** [livewire/livewire#4536] [4. General]
Hi folks 👋

Anyone already working on a pull request for Laravel 9 compatibility?
I've seen that `calebporzio/sushi` needs to be updated and I've created a pull request for that: https://github.com/calebporzio/sushi/pull/103

But it seems more stuff is broken. My local installations fails with the following error message and I have no idea how to fix that.
(I assume it's connected with the bump from Symfony 5 to Symfony 6 components in Laravel's internals)

```shell
Tests\Unit\TestableLivewireCanAssertNoRedirectTest::can_assert_no_redirect_will_fail_if_redirected
Failed asserting that exception of type "Symfony\Component\HttpFoundation\Exception\SessionNotFoundException" matches expected exception "PHPUnit\Framework\AssertionFailedError". Message was: "Session has not been set." at
/vendor/symfony/http-foundation/Request.php:702
/src/Controllers/HttpConnectionHandler.php:67
/src/Controllers/HttpConnectionHandler.php:32
/livewire/src/Controllers/HttpConnectionHandler.php:16
```

(I maintain [wnx/laravel-stats](https://github.com/stefanzweifel/laravel-stats/pull/202) and wanted to make that package compatible with Laravel 9 before its release next week.)

**Comment by joshhanley** [comment]
@stefanzweifel yeah I've got a Laravel 9 PR ready to go, just waiting for orchestral/testbench and orchestral/testbench-dusk to be tagged with Laravel 9 support, so we can run our full test suite and fix any issues.

Thanks for the Sushi PR! I will see if Caleb can have a look when he gets the chance 🙂

**Reply by joshhanley** [reply]
@stefanzweifel looks like Sushi will also need to wait for testbench to be tagged

**Reply by stefanzweifel** [reply]
> @stefanzweifel looks like Sushi will also need to wait for testbench to be tagged

Haven't locked at testbench-dusk, but Suhsi's testsuite can already be executed for Laravel 9.

https://github.com/stefanzweifel/sushi/actions/runs/1704392420

First party packages have already been aliased for Laravel 9.

**Reply by joshhanley** [reply]
@stefanzweifel oh sweet! Yeah it looks like there was a branch of testbench created last week after I was looking at it with Laravel 9 support.

Been working on our support yesterday and today but running into a few bugs with the upstream packages https://twitter.com/_joshhanley/status/1483555925877456898?s=20
